tools/cli/internal/openapi/filter/hidden_envs.go

```diff
@@ -130,6 +147,19 @@ func (f *HiddenEnvsFilter) isResponseHiddenForEnv(response *openapi3.ResponseRef
 	return false
 }
 
+func (f *HiddenEnvsFilter) isContentTypeHiddenForEnv(contentType *openapi3.MediaType) bool {
+	if contentType == nil {
+		return false
+	}
+
+	if extension, ok := contentType.Extensions[hiddenEnvsExtension]; ok {
```

Comment:
do we need to do the contentType == nil or contentType.Extensions will handle it returning !ok ?

Reply:
we do or we'd get a panic on contentType.Extensions, right?